packages/dcent/src/index.ts

```diff
@@ -110,6 +110,7 @@ function dcent({
 
           const provider = new StaticJsonRpcProvider(currentChain.rpcUrl)
 
+          // @ts-ignore
```

Comment:
Were you getting type errors here? I am not seeing any type errors on this line that requires ts-ignore

Reply:
I am indeed getting a type error there. Also, when building the package the dcent package produced an error that prevented the build process from succeeding.
But maybe it's just on my machine this is occurring since it's working with yours. I removed that line.

Reply:
Ah yeah that's weird. I'll take another look at that as a separate issue.